tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx

```diff
@@ -2,19 +2,24 @@ import { useBackend } from '../backend';
 import { Button, Section } from '../components';
 import { Window } from '../layouts';
 
+type Data = {
+  priority: string[];
+  minor: string[];
+};
+
 export const AtmosAlertConsole = (props, context) => {
-  const { act, data } = useBackend(context);
-  const priorityAlerts = data.priority || [];
-  const minorAlerts = data.minor || [];
+  const { act, data } = useBackend<Data>(context);
+  const { priority = [], minor = [] } = data;
+
   return (
     <Window width={350} height={300}>
       <Window.Content scrollable>
         <Section title="Alarms">
           <ul>
-            {priorityAlerts.length === 0 && (
-              <li className="color-good">No Priority Alerts</li>
+            {priority.length === 0 && (
+              <li className="color-good">No Priority </li>
```

Comment:
Replace fail?

Reply:
They had just renamed the alerts in order to cast it to an array, this is unnecessary

Reply:
And what does that have to do with you removing "Alerts" from this text

Reply:
That is not what the ui data is called

Reply:
![image](https://user-images.githubusercontent.com/35135081/175878878-c36ae28d-2fde-4d7b-8d23-abeb9064ee5f.png)

Where did this go jlsnow301

Reply:
Mothblocks refers to the fact that you modified the user-visible text below as well (to "No priority " with a trailing space, too)

Reply:
Oh oops! On it